x/superfluid/client/cli/tx.go

```diff
@@ -31,8 +31,6 @@ func GetTxCmd() *cobra.Command {
 		NewSuperfluidUndelegateCmd(),
 		NewSuperfluidUnbondLockCmd(),
 		// NewSuperfluidRedelegateCmd(),
-		NewCmdSubmitSetSuperfluidAssetsProposal(),
-		NewCmdSubmitRemoveSuperfluidAssetsProposal(),
```

Comment:
Can we also delete the cli txs?

Ty for this!

Reply:
no we cant delete it, `NewProposalHandler` requires a CLI-Handler https://github.com/osmosis-labs/osmosis/blob/main/x/superfluid/client/proposal_handler.go#L11

Reply:
ah I see, your right!